app/controllers/admin/organizations_controller.rb

```diff
@@ -65,5 +79,11 @@ def search(params)
         end
       end
 
+      def soft_delete_organization_user(organization)
```

Comment:
Why not put this method/logic in the user model? so it would be called as user.soft_destroy

Reply:
Seemed more reasonable (at first sight) to include it within `Admin::OrganizationsController` since it's tightly related to it, but I guess you're right.